### PR TITLE
Allow for different demo snippets to be selected

### PR DESCRIPTION
**Pull request by coolreader18**
The selection works, but I'm looking for ideas for what other snippets could be put there. Everything's stored in `wasm/demo/snippets/*.py`; a separate demo snippet would mean a new `.py` file in that directory.

### CODE REVIEW

**Comment by windelbouwman** [comment]
You can check this issue, it contains mandelbrot.py:

https://github.com/RustPython/RustPython/issues/352

File: wasm/demo/snippets/fibonacci.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+n1 = 0
+n2 = 1
+count = 0
+until = 10
+
+print(f"These are the first {until} numbers in the Fibonacci sequence:")
```
Comment (windelbouwman) [inline wasm/demo/snippets/fibonacci.py]: Sweet! f-strings!

**Comment by coolreader18** [comment]
It takes a while to run, for me about 20 seconds, and the page is frozen during that time. Still a good example, though.

**Comment by coolreader18** [comment]
I think it could be improved by a requestAnimationFrame call, but that would be for another PR to add that as a builtin.

**Pull request merged**